miso-web/src/main/webapp/WEB-INF/pages/editStudy.jsp

```diff
@@ -37,113 +37,43 @@
-            <c:otherwise><i>Unsaved</i></c:otherwise>
-          </c:choose>
-        </span></td>
-      </tr>
-      <tr>
-        <td class="h">Project Name:</td>
-        <td>
-        <!-- When opening a new study without a known project ID (i.e., from the Studies page), dropdown should display project name when
-        in plain sample mode, shortname when in detailed sample mode. ${projects} will come in sorted based on these attributes from
-        EditStudyController. -->
-        <c:choose>
-          <c:when test="${not empty study.project}">
-            <input type="hidden" value="${study.project.id}" name="project" id="project"/>
-            <a href='<c:url value="/miso/project/${study.project.id}"/>'><span id="projectName">${study.project.name}</span></a>
-          </c:when>
-          <c:otherwise>
-            <miso:select id="project" path="project" items="${projects}" itemLabel="${detailedSample? \"shortName\" : \"name\" }" 
-              itemValue="id"/>
-          </c:otherwise>
-        </c:choose>
-        </td>
-      </tr>
-      <tr>
-        <td>Name:</td>
-        <td><span id="name">
-          <c:choose>
-            <c:when test="${study.id != 0}">${study.name}</c:when>
-            <c:otherwise><i>Unsaved</i></c:otherwise>
-          </c:choose>
-        </span></td>
-      </tr>
-      <tr>
-        <td class="h">Alias:</td>
-        <td><form:input id="alias" path="alias" class="validateable"/><span id="aliascounter" class="counter"></span></td>
-      </tr>
-      <tr>
-        <td>Description:</td>
-        <td><form:input id="description" path="description" class="validateable"/><span id="descriptioncounter" class="counter"></span>
-        </td>
-      </tr>
-      <c:if test="${not empty study.accession}">
-        <tr>
-          <td class="h">Accession:</td>
-          <td><a href="http://www.ebi.ac.uk/ena/data/view/${study.accession}"
-                 target="_blank">${study.accession}</a>
-          </td>
-        </tr>
-      </c:if>
-      <tr>
-        <td>Study Type:</td>
-        <td><miso:select id="studyType" path="studyType" items="${studyTypes}" itemLabel="name" itemValue="id" /></td>
-      </tr>
-      </table>
-      <br/>
-      
-      <script type="text/javascript">
-        jQuery(document).ready(function () {
-          // Attaches a Parsley form validator. 
-          Validate.attachParsley('#study-form');
+            <div class="breadcrumbspopup">
+                ${study.project.name}
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div class="sectionDivider" onclick="Utils.ui.toggleLeftInfo(jQuery('#note_arrowclick'), 'notediv');">Quick Help
+      <div id="note_arrowclick" class="toggleLeft"></div>
+    </div>
+    <div id="notediv" class="note" style="display:none;">A Study contains more fine-grained information about
+      the sequencing Project. Studies can contain any number of sequencing Experiments and Analysis.
```

Comment:
"Studies can contain any number of [...] Analysis"

Analysis -> Analyses?

Reply:
fixed